build/build.js

```diff
@@ -60,4 +61,27 @@ function write(dest, code) {
 }
 function logError(e) {
   console.log(e)
+}
+
+function copyHooks() {
```

Comment:
I added this function in order to ensure the hooks directory will be copied to `dist` folder on build step. And this way we'll ensure the hooks directory will be included in built package.
But I am not sure what is exactly the approach you use to pack/bundle the plugin's code.

We need this code in case when we want to have all the code that will be bundled inside `dist` folder.
If you think this is unneeded, this function can be removed. If we decide to remove it, this line should be changed https://github.com/nativescript-vue/nativescript-vue/pull/361/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R9.